pkg/controller/controller_binding_test.go

```diff
@@ -242,7 +242,7 @@ func TestReconcileServiceBindingNonExistingClusterServiceClass(t *testing.T) {
 	events := getRecordedEvents(testController)
 	assertNumEvents(t, events, 1)
 
-	expectedEvent := corev1.EventTypeWarning + " " + errorNonexistentClusterServiceClassMessage + " " + "ServiceBinding \"test-ns/test-binding\" references a non-existent ClusterServiceClass \"" + testNonExistentClusterServiceClassName + "\""
+	expectedEvent := corev1.EventTypeWarning + " " + errorNonexistentClusterServiceClassMessage + " " + "ServiceBinding \"test-ns/test-binding\" references a non-existent ClusterServiceClass \"" + testNonExistentClusterServiceClassName + "\" : K8S name \"\""
```

Comment:
not blocking for this PR; noting here for the future - this is a hard-to-read string, and I think it should be written in a `fmt.Sprintf` instead